x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/system_prompt/system_prompt_modal/system_prompt_settings.tsx

```diff
@@ -100,11 +104,21 @@ export const SystemPromptSettings: React.FC<Props> = React.memo(
     const handleConversationSelectionChange = useCallback(
       (currentPromptConversations: Conversation[]) => {
         const currentPromptConversationIds = currentPromptConversations.map((convo) => convo.id);
+        const getDefaultSystemPromptId = (convo: Conversation) =>
```

Comment:
Looks like default conversation selection has a bug where it'll auto-select multiple conversations:

<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/14bda840-98e2-4c14-baa1-9ff1464dcd37" />
</p>

Reply:
Fixed